kovri.i2p/_i18n/fr/contributing.md

```diff
@@ -1,72 +1,73 @@
+## Assurance qualité
+- Voir notre guide d'[Assurance Qualité](https://github.com/monero-project/kovri/blob/master/doc/QUALITY_ASSURANCE.md) pour en savoir plus.
 
-## Compliance
-- We aim for complete C++11/14 compliance; feel free to use this to your advantage with your work
-- It is also highly recommended to use the standard library and dependency libraries when possible
 
-## Sending your work
-To contribute your work, please proceed with the following:
 
-1. Fork Kovri
-2. Read our [style guide](https://github.com/monero-project/kovri/blob/master/doc/STYLE.md)
-3. Create a [topic branch](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging)
-4. [**Sign**](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work) your commit(s)
-5. Send a pull-request to branch ```master```
-   - We currently do not have any tags as we are in pre-alpha. For now, you can base your work off of master.
-   - Commit messages should be verbose by default, consisting of a short subject line (50 chars max), a blank line, and detailed explanatory text as separate paragraph(s) - unless the title alone is self-explanatory.
-   - Commit title should prepend class or aspect of project. For example, "HTTPProxy: implement User-Agent scrubber. Fixes #193." or "Garlic: fix uninitialized padding in ElGamalBlock".
-   - If a particular commit references another issue, please add a reference. For example "See #123", or "Fixes #123". This will help us resolve tickets when we merge into ```master```.
-   - In general, commits should be [atomic](https://en.wikipedia.org/wiki/Atomic_commit#Atomic_commit_convention) and diffs should be easy to read. For this reason, please try to not mix formatting fixes with non-formatting commits.
-   - The body of the pull request should contain an accurate description of what the patch does and provide justification/reasoning for the patch (when appropriate). You should include references to any discussions such as other tickets or chats on IRC.
+## Conformité
+- Nous ésperons compléter la conformité C++11/14; n'hésitez pas à l'utiliser pour vous faciliter la tâche dans votre travail
+- Il est aussi vivement recommandé d'utiliser la librairie standard et les bibliothèques de dépendances si vous le pouvez
+
+## Envoyer votre travail
+Pour mettre à contribution votre travail, veuillez procéder de la façon suivante:
 
-## Proposals
-To contribute a proposal, please review our [open issues](https://github.com/monero-project/kovri/issues) for existing proposals. If what you propose is not there, then [open a new issue](https://github.com/monero-project/kovri/issues/new).
+1. Fork Kovri
+2. Lisez notre [style guide](https://github.com/monero-project/kovri/blob/master/doc/STYLE.md)
+3. Creez une [branche pour le sujet concerné](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging)
+4. [**Signez**](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work) votre ou vos commit(s)
```

Comment:
It's not about translating the link. It just turns out that git have a french translation of this exact doc.